files/fr/web/css/fit-content/index.md

```diff
@@ -38,8 +38,6 @@ block-size: fit-content;
 </div>
 ```
 
-#### CSS
-
```

Comment:
In order to maintain the semantics of the pages between the Original version and the FR Translation, it will be necessary to keep the titles that have been placed on https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content#examples